.npmrc

```diff
@@ -0,0 +1 @@
+package-lock=false
```

Comment:
fwiw I do this in my ~/.npmrc so it's not normally required per-project because I'm the publisher, but this is fine